### ISSUE TITLE
Warfront related quests done wrong

### ISSUE THREAD

**Issue by yoshimo** [smaitch/Grail#59]
Quests that belong to arathi highlands are shown in the harbor of the horde as available.
They are also shown when the arathi highlands belong to the alliance.
Similar issues arise with rhe contribution quests for war fronts.

We need to figure out a way to hide the quests when they aren't offered in game

**Comment by smaitch** [comment]
Yes, I do not know what API to use to determine when those dailies, etc. are available.

**Comment by yoshimo** [comment]
Maybe https://www.curseforge.com/wow/addons/contributioneta/ can help with that.
